cli/popper/commands/cmd_run.py

```diff
@@ -133,3 +149,38 @@ def run_pipeline(action, wfile, skip, workspace, reuse,
         log.info('Action "{}" finished successfully.'.format(action))
     else:
         log.info('Workflow finished successfully.')
+
+
+def workflows_from_commit_message(workflows):
+    msg = scm.get_head_commit_message()
+
+    if 'Merge' in msg:
+        log.info("Merge detected. Reading message from merged commit.")
+        commit_id = re.search(r'Merge (.+?) into', msg).group(1)
```

Comment:
a merge commit consists of a commit A being merged into B. For example, [this one](https://github.com/systemslab/popper/commit/c4d532d6e3bba7261f7f6bd60b4243a42398f4fa). The idea is to obtain the commit message for A. Since we are now using GitPython, we might be able to use it to obtain this information (as opposed to using regular expressions.

Reply:
Thanks !